docs/advanced/experimental/Kubernetes.md

```diff
@@ -0,0 +1,132 @@
+# Cookbook on `Kubernetes` Deployment
+
+Deploy your `Flow` on `Kubernetes`.
+## Requirements
+- please setup a `Kubernetes` cluster and pull the credentials. e.g.:
+  [minikube](https://minikube.sigs.k8s.io/docs/start/) (testing),
+  [Google Kubernetes Engine](https://cloud.google.com/kubernetes-engine),
+  [Amazon EKS](https://aws.amazon.com/eks),
+  [Azure Kubernetes Service](https://azure.microsoft.com/en-us/services/kubernetes-service),
+  [Digital Ocean](https://www.digitalocean.com/products/kubernetes/)
+  
+## Deploy your `Flow`
+To deploy a `Flow` on `Kubernetes`, you have to set the `infrastructure='K8S'` when creating the `Flow`.
+Call `flow.start()` to trigger the deployment. Make sure all `Executors` are containerized.
+
+## Example
+In this example, we assume that, there is a postgres database deployed on `Kubernetes` already. 
+You can also modify the database configuration to use a postgres database outside the `Kubernetes` cluster.
+Deploy the index `Flow` on `Kubernetes`:
+```python
+from jina import Flow
+
+flow = Flow(
+    name='index-flow', port_expose=8080, infrastructure='K8S', protocol='http'
```

Comment:
very minor: I would try to keep the exampleas as minimal as possible? So if there is no particular reason for http (like showing a curl command or so), I dont see the point of specifying this here?

Reply:
Okay, I also added another minimal example.